.github/workflows/build.yml

```diff
@@ -43,6 +43,8 @@ jobs:
 
       - name: Setup .NET Core SDK
         uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 8.0.100-preview.1.23115.2
```

Comment:
Extract the SDK version to a variable like for Npgsql, since it appears multiple times?

Reply:
I'll also add it in codeql-analysis.yml (I've added it in Npgsql's too).
I think it is a good idea to consistently have the SDK version to use in one place somewhere near the top of the file.